utils.py

```diff
@@ -71,6 +71,9 @@ def wrapper_validate(*args, **kwargs):
             openapi_request = FlaskOpenAPIRequest(request)
             result = validator.validate(openapi_request)
             if result.errors:
+                error_response = (
```

Comment:
Changing the error in which this code is executed will not impact the overall behavior since we are already checking for the mime-type before. In this way, we leave the error message more generic and augment it if the validator provides additional info.